framework/src/problems/FEProblemBase.C

```diff
@@ -1461,6 +1461,9 @@ FEProblemBase::setVariableAllDoFMap(const std::vector<const MooseVariableFEBase
 void
 FEProblemBase::prepare(const Elem * elem, THREAD_ID tid)
 {
+  if (!elem->has_quadrature_support())
+    return;
```

Comment:
These breaks are kind of interesting.  Would skipping irrelevant initialization mean you'd be able to work with these via a Kernel rather than a UserObject?